chrome.manifest

```diff
@@ -1 +1,2 @@
+content usearch chrome/
```

Comment:
:+1: I'll change that. I can also change the mapping here and the xml file name to avoid "content/content/content.xml" in the chrome URL, which seems suboptimal